pig/pom.xml

```diff
@@ -159,6 +159,15 @@ limitations under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.owasp</groupId>
+        <artifactId>dependency-check-maven</artifactId>
+        <executions>
+          <execution>
+            <phase>none</phase>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
Can we make it warn instead of fail the build for these modules? That would at least still show what vulnerabilities there are. In case someone wants to tackle it.

Reply:
Think warning would be better than straight skipping. Not sure what the output would look like though and if it would be misleading

Reply:
Thanks @risdenk I guess you are right; I have another workaround, will commit shortly.